frontend/views/components/VotingSystemInput.vue

```diff
@@ -0,0 +1,95 @@
+<template lang='pug'>
+  div
+    slider-continuous.c-slider(
+      :class='ephemeral.sliderClass'
+      :uid='type'
+      :label='config[type].slideLabel'
+      :min='config[type].slideMin'
+      :max='config[type].slideMax'
+      :unit='config[type].slideUnit'
+      :value='value'
+      @input='handleInput'
+    )
+    // [1] - inner el has paddings. A wrap is needed for a smooth transition.
+    transition-expand
+      div(v-if='warnMajority') <!-- [1] -->
+        banner-simple.c-banner(severity='warning')
+          i18n The percentage value you are choosing is most likely too low for a decision that can have a potentially significant impact  on a person's life. Please consider using a
+          | &nbsp;
+          i18n(
+            tag='a'
+            class='link'
+            href='https://groupincome.org/2016/09/deprecating-mays-theorem/#when-majority-rule-can-harm'
+            target='_blank'
+          ) supermajority threshold.
```

Comment:
Sentences must never be split like this, it makes translation impossible.

Please use `{a_} ... {_a}` instead

Reply:
True. The way it was before it made sense because it was 2 different sentences. Fixed.